src/Core/Settings.h

```diff
@@ -1080,6 +1080,7 @@ class IColumn;
     M(String, format_schema, "", "Schema identifier (used by schema-based formats)", 0) \
     M(String, format_template_resultset, "", "Path to file which contains format string for result set (for Template format)", 0) \
```

Comment:
Let's also add similar setting for resultset template, will be useful to have both

Reply:
Also, please, add new settings to `SettingsChangesHistory.h`